pkg/api/http_server.go

```diff
@@ -54,7 +54,7 @@ type HTTPServer struct {
 	httpSrv     *http.Server
 	middlewares []macaron.Handler
 
-	RouteRegister        routing.RouteRegister            `inject:""`
+	RouteRegister        routing.RouteRegister
```

Comment:
I don't think we can just stop injecting the RouteRegister without also fixing all the locations where it is modified. We use it in Enterprise to extend the API and it's injected in a few places in OSS too, I didn't look into what it was used for. But I presume having the http_server use a different instance than the rest of Grafana will cause problems.

Reply:
@xlson Aha, thanks for making me aware of that.

Reply:
@xlson Reverted this change.